### ISSUE TITLE
new manual for 3rd party plugins

### ISSUE THREAD

**Issue by tjingboem** [csound/manual#748]
https://github.com/csound/plugins :
these are all 3rd party Csound plugins.
It needs its own separate manual and many opcodes, that are now part of the Canonical Manual, are to be transferred over to this new manual. 

Thus leaving the Canonical Manual the manual pages of opcodes that come with a freshly installed Csound.

**Comment by tjingboem** [comment]
- [ ] AbletonLinkOpcodes
- [ ] Bela
- [ ] CUDA
- [ ] chua
- [ ] faustcsound
- [ ] fluidOpcodes
- [ ] hdf5
- [ ] image
- [ ] jackops
- [ ] linear algebra
- [ ] mp3
- [ ] opencl
- [ ] p5glove
- [ ] py opcodes
- [ ] STK opcodes
- [ ] websockets
- [ ] widgets
- [ ] WIImote

**Comment by csounder** [comment]
a lot of things. Dr. Richard BoulangerProfessorElectronic Production and DesignBerklee College of MusicOn Jun 14, 2024, at 2:28 PM, Menno Knevel ***@***.***> wrote:﻿
 
 
  
  
    AbletonLinkOpcodes
    Bela
    CUDA
    chua
    faustcsound
    fluidOpcodes
    hdf5
    image
    jackops
    linear algebra
    mp3
    opencl
    p5glove
    py opcodes
    STK opcodes
    websockets
    widgets
    WIImote

**Comment by tjingboem** [comment]
and probably more to come...not yet clear

**Comment by csounder** [comment]
I wonder if the Risset things should be in here too?

*- Dr.B*


*Dr. Richard Boulanger*

Professor

Electronic Production and Design

*Berklee College of Music*

Professional Writing & Technology Division

**Comment by tjingboem** [comment]
can you give me a link of these Risset things?

**Comment by csounder** [comment]
Maybe this:   https://github.com/csound-plugins/risset

*- Dr.B*


*Dr. Richard Boulanger*

Professor

Electronic Production and Design

*Berklee College of Music*

Professional Writing & Technology Division

**Comment by csounder** [comment]
His Docs and examples are especially great.  However, both I and John
ffitch have had difficulty installing them and getting them to run.
- hopefully you will have luck.  I would have loved to see many of these as
actual opcodes in Csound.

*- Dr.B*


*Dr. Richard Boulanger*

Professor

Electronic Production and Design

*Berklee College of Music*

Professional Writing & Technology Division

**Comment by csounder** [comment]
https://csound-plugins.github.io/risset-docs/

*- Dr.B*


*Dr. Richard Boulanger*

Professor

Electronic Production and Design

*Berklee College of Music*

Professional Writing & Technology Division

**Comment by tjingboem** [comment]
The repos of Eduardo already have their own documentation: https://csound-plugins.github.io/risset-docs/

**Comment by tjingboem** [comment]
i agree that these opcodes are great. And i have them installed. 
I think we could call them 4th party plugins...

Also, Rory has a few opcodes in his repo: https://github.com/rorywalsh/csoundPluginOpcodes/tree/master

I was thinking of adding to the Canonical Csound Reference Manual some words and links about these repos

**Comment by csounder** [comment]
Yes - a good plan as these are important and useful 'missing' pieces that
most would want to install.

*- Dr.B*


*Dr. Richard Boulanger*

Professor

Electronic Production and Design

*Berklee College of Music*

Professional Writing & Technology Division
